oaknational/oak-ai-lesson-assistant
# feat(feedback): chat and message level feedback

## Description

A combination of
- @tomwisecodes 's work adding feedback functionality to on lesson plan sections
- my message id fixes

## Issue(s)

- [In-app feedback](https://www.notion.so/oaknationalacademy/Implement-in-app-feedback-634596c3e5d844f789ccd8b58ba206ac?pvs=4)
- [Message id fixes](https://www.notion.so/oaknationalacademy/Message-ID-fixes-c224675465ef44aeb3fe3e1bbc71b30a?pvs=4)

## How to test

1. Go to https://oak-ai-lesson-assistant-73pc5tqpt.vercel-preview.thenational.academy
2. Click on \_\_\_\_\_\_
3. You should see \_\_\_\_\_\_

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
